src/js/game/Entities/BaseEntity.js

```diff
@@ -651,4 +653,17 @@ module.exports = class BaseEntity {
     return [offset[0] * -1, offset[1] * -1];
   }
 
+  handleMoveOffIronDoor(moveOffset) {
+    const formerPosition = [this.position[0] + moveOffset[0], this.position[1] + moveOffset[1]];
+    if (!this.controller.levelModel.inBounds(formerPosition[0], formerPosition[1])) {
```

Comment:
I'm just now realizing we have both `levelModel.inBounds(x, y)` and `levelPlane.inBounds(position)`.  Not something we need to clean up for this PR, but something we should address in the future!